### PR TITLE
Add info about new employee tech stipend options

### PR DESCRIPTION
**Pull request by jrozra200**
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/tech-stipend-update/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/040-employee-handbook-us/tech-stipend.md [hunk 1]
```diff
@@ -21,3 +21,5 @@ As part of CivicActions commitment to work/life balance and our movement toward
...
 
 The Technology Stipend will be paid at the current reimbursement Payment Amount as determined by CivicActions. The Payment Amount will be direct deposited using the same method as the team member's paycheck and other expense reimbursements. To ensure compliance with federal law, IRS requires, and state and local laws, CivicActions may in its sole discretion deducted and/or convert to a taxable payment a prorated amount of the last Technology Stipend payment made to an employee who leaves CivicActions employment in the year following the Eligibility Date. Part Time Eligible Team Members employees will receive a proportional amount based on the average number of hours scheduled to work per month.
+
+[^1]: If you or one of your employees are within the ninety-day Introductory Period, do not have equipment that is satisfactory to complete your defined role, and would benefit from recieving your Technology Stipend early, there are options available to you. Reach out to your manager and/or a people operations professional to find out more information.
```
Comment (dmundra) [inline docs/040-employee-handbook-us/tech-stipend.md]: Thanks @jrozra200. Tiny typo, 'recieving' => 'receiving'

Comment (jrozra200) [reply docs/040-employee-handbook-us/tech-stipend.md]: just fixed it!

**Comment by jrozra200** [comment]
@dmundra - I'm not sure I am understanding what is failing now? Can you help me understand how to fix the remark-lint failure?

**Comment by dmundra** [comment]
> @dmundra - I'm not sure I am understanding what is failing now? Can you help me understand how to fix the remark-lint failure?

@jrozra200 if you jump to 'Files changed' tab in GitHub you can see the remark-lint errors. It is complaining about `undefined definition` for some reason and read the docs did not generate a version we can review so 🤷🏾

**Comment by jrozra200** [comment]
@dmundra - looks like I can't hyperlink to the comment below. Let me know if this works for you.

**Comment by dmundra** [comment]
> @dmundra - looks like I can't hyperlink to the comment below. Let me know if this works for you.

@jrozra200 looks like I found a footnote example https://github.com/readthedocs/sphinx_rtd_theme/blob/master/docs/demo/demo.rst#footnotes for readthedocs, do you want to try that?

**Comment by dmundra** [comment]
@jrozra200 we can ignore the remark-lint and merge and see how this looks on handbook.

**Review by dmundra — APPROVED** [review]

**Comment by mgifford** [comment]
Why is the docs/050-how-we-work/tools/github.md page edited here? This isn't related to the tech stipend. 

Think it might conflict or overwrite what is here https://github.com/CivicActions/handbook/pull/829

**Comment by dmundra** [comment]
> Why is the docs/050-how-we-work/tools/github.md page edited here? This isn't related to the tech stipend.
> 
> Think it might conflict or overwrite what is here #829

@mgifford it is a side-effect of merging changes and auto-rebasing this branch. Sometimes commits from other merges appear in here with the pre-commit. I think this should merge fine when it is approved.

File: docs/040-employee-handbook-us/tech-stipend.md [hunk 2]
```diff
@@ -21,3 +21,7 @@ As part of CivicActions commitment to work/life balance and our movement toward
...
 
 The Technology Stipend will be paid at the current reimbursement Payment Amount as determined by CivicActions. The Payment Amount will be direct deposited using the same method as the team member's paycheck and other expense reimbursements. To ensure compliance with federal law, IRS requires, and state and local laws, CivicActions may in its sole discretion deducted and/or convert to a taxable payment a prorated amount of the last Technology Stipend payment made to an employee who leaves CivicActions employment in the year following the Eligibility Date. Part Time Eligible Team Members employees will receive a proportional amount based on the average number of hours scheduled to work per month.
+
+<sup>1</sup>: If you or one of your employees are within the ninety-day Introductory Period, do not have equipment that is
```
Comment (thejuliekramer) [inline docs/040-employee-handbook-us/tech-stipend.md]: replace employee with team member

**Comment by jrozra200** [comment]
@thejuliekramer and @elizabethraley - updates ready for your review.

**Comment by dmundra** [comment]
> Why is the docs/050-how-we-work/tools/github.md page edited here? This isn't related to the tech stipend.
> 
> Think it might conflict or overwrite what is here #829

@mgifford rebasing again removed the extra changes so ya just a rebasing oddity is what you noticed.

**Review by elizabethraley — APPROVED** [review]

**Review by marctjones — CHANGES_REQUESTED** [review]
Jake,

Thanks for this suggestion. Can you cancel this PR request, and I will look at incorporating the spirit of this change into the handbook.

**Comment by marctjones** [comment]
Closing PR. But noting the wording change suggested. I will look into incorporating the suggested wording change into the handbook.

**Pull request closed without merging**